buildroot/bin/restore_configs

```diff
@@ -13,21 +13,11 @@ for file in files_to_remove:
   if os.path.exists(file):
     os.remove(file)
 
-def use_example_configs():
-  try:
-    subprocess.run(['use_example_configs'], check=True)
```

Comment:
This uses the instance of `use_example_configs` found in your $PATH, which for some rare cases might be desirable. Let's preserve that behavior.